### ISSUE TITLE
Toe hyperpigmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11690]
# Toe hyperpigmentation

## Proposed definition

Increased pigmentation of the skin of the toes.

## Parent term

- `HP:0000953 Hyperpigmentation of the skin`

## Synonyms

- Hyperpigmentation of the toes

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001528
- label: Toe hyperpigmentation
- parents: [Hyperpigmentation of the skin (HP:0000953)](https://hpo.jax.org/app/browse/term/HP:0000953)
- pmid: 
- def: Increased pigmentation of the skin of the toes.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11690
- synonyms:
